README.md

```diff
@@ -106,6 +106,30 @@ In very high throughput environments it is possible to improve performance by ch
 - `sysctl -w net.unix.max_dgram_qlen=X` - Set datagram queue size to X (default value is usually 10).
 - `sysctl -w net.core.wmem_max=X` - Set the max size of the send buffer for all the host sockets.
 
+### Maximum packets size in high-throughput scenarios
+
+In order to have the most efficient use of this library in high-throughput scenarios,
+default values for the maximum packets size have already been set for both UDS (8192 bytes)
+and UDP (1432 bytes) in order to have the best usage of the underlying network.
```

Comment:
I don't think we use the UDS default yet. It's something related to a limitation in the agent.

Reply:
You're right I forgot about this, I'll adapt the doc